components/eam/cime_config/config_component.xml

```diff
@@ -3,8 +3,8 @@
 <?xml-stylesheet type="text/xsl" href="definitions_variables" ?>
 
 <!DOCTYPE phys_defaults [
-  <!ENTITY eam_phys_defaults "-nlev 72 -clubb_sgs -microphys mg2">
-  <!ENTITY eam_chem_defaults "-chem linoz_mam4_resus_mom_soag -rain_evap_to_coarse_aero">
+  <!ENTITY eam_phys_defaults "-nlev 72 -clubb_sgs -microphys p3">
+  <!ENTITY eam_chem_defaults "-chem superfast_mam5_resus_mom_soag -rain_evap_to_coarse_aero -vbs -usr_mech_infile $SRCROOT/components/eam/chem_proc/inputs/pp_chemUCI_linozv3_mam5_vbs.in">
```

Comment:
In the final v3 version, we would like to create a new "-chem" option, so 1) the "-usr_mech_infile" option will be replaced by the pp_* directory and 2) the superfast mechanism will be reenabled. This can be done in a separate future PR.

Reply:
Agreed, @tangq . Let's keep that in mind.

Reply:
I would suggest using a different chem package name such as chemUCI_linozv3_mam5_vbs. This would also need to change configure, phys_control.F90 and perhaps model_aer_opt.F90. xx_resus_mom_soag is inherited from v1/v2 chem packages.

Reply:
I know all these chem options are tangled together with various interdependencies, but would it make more sense to just have flags like "-mam5" and "-uci_chem" to turn each feature on and then enforce the dependencies elsewhere, like in eam/bld/configure?

And shouldn't that "usr_mech_infile" be a namelist variable?

Reply:
The specification of usr_mech_infile in cam_config_opts is tentative. As @tangq said, it will be worked out to use a single `-chem` option to automatically select the mechanism file. It is not a namelist variable. It is an option to eam/bld/configure, which would then invoke chem preprocessor to parse and generate Fortran codes for use by the model. Agreed we should introduce some simple flags like you suggested to control the logics in configure and build-namelist to replace some less flexible treatments (e.g., parsing for chemUCI-Linoz subtring in use_case file name to order to specify needed species).